brands/shop/swimming_pool.json

```diff
@@ -0,0 +1,12 @@
+{
+  "shop/swimming_pool|Leslie's Pool Supplies": {
```

Comment:
The full name is Leslie's Swimming Pool Supplies.  I'm not sure if we want that as a `name` or `official_name`

Reply:
I don't think that's actually their full name anymore. It used to be in the logo, but the new logos say "pool supplies service & repair", which is what the website uses when they refer to their full name:

This:

![image](https://user-images.githubusercontent.com/1015200/60403137-3e826c80-9b4e-11e9-853f-01bce2421171.png)


vs this:

![image](https://user-images.githubusercontent.com/1015200/60403139-4b06c500-9b4e-11e9-87eb-e1280b89487d.png)

Reply:
Good catch!  Carry on

Reply:
It must be pretty recent because I swear the last one I went to had the old name.